src/WfsFilterUtil/WfsFilterUtil.ts

```diff
@@ -61,31 +63,32 @@ class WfsFilterUtil {
    * Creates a filter for a given feature type considering configured
    * search attributes, mapped features types to an array of attribute details and the
    * current search term.
-   * Currently, supports EQUALTO and LIKE filters only, which can be combined with
+   * Currently, supports EQUAL_TO and LIKE filters only, which can be combined with
    * OR filter if searchAttributes array contains multiple values though.
    *
+   * @param featureType
    * @param {string} searchTerm Search value.
    * @param attributeDetails
    *   attributes that should be searched through.
-   * @return {OlFormatFilter} Filter to be used with WFS GetFeature requests.
+   * @return {OlFilter} Filter to be used with WFS GetFeature requests.
    * @private
    */
   static createWfsFilter(
     featureType: string,
     searchTerm: string,
     attributeDetails: AttributeDetails
-  ): OlFormatFilter | null {
+  ): OlFilter | undefined {
```

Comment:
isn't this one a breaking change? (method return type is changed)

Reply:
I renamed `OlFormatFilter` to `OlFilter`, so here's no breaking change imho. On the other hand, I agree that switch of `null` to `undefined` is rather a breaking change. I'll change the commit message accordingly.

Reply:
Please check again @annarieger 🙏🏼